editors/linux-wps-office/Makefile

```diff
@@ -0,0 +1,55 @@
+PORTNAME=	wps-office
+DISTVERSION=	11.1.0.10161
+CATEGORIES=	editors linux
+MASTER_SITES=	https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/10161/
+PKGNAMEPREFIX=	linux-
+DISTNAME=	${PORTNAME}-${PORTVERSION}.XA-1.x86_64
+
+MAINTAINER=	ygy@FreeBSD.org
```

Comment:
Just looking at https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=252736
Doesn't original submitter want to be the maintainer?